### PR TITLE
Update getmetadatapropertyvalue.md

### PR DESCRIPTION
**Pull request by an-emma**
Added one known limitation.

### CODE REVIEW

**Comment by PRMerger13** [comment]
@an-emma : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: stream-analytics-docs/getmetadatapropertyvalue.md [hunk 1]
```diff
@@ -89,3 +89,5 @@ GetMetadataPropertyValue has the following limitations of usage:
 * Using `SELECT *` in your query causes duplicate columns. To prevent duplicate columns, list columns individually in your SELECT statement.
 
 * The alias you give your Metadata Property Value will be lowercase regardless of the casing used in your query. For example, `SELECT GetMetadataPropertyValue(ehInput, 'EventId') AS eventPrimaryKey` outputs as `eventprimarykey`. To preserve casing, use compatibility level 1.2.
+
+* This function does not work in portal preview results pane or Visual Studio/VS Code local query testing. It only works when job is running in public cloud.
```
Comment (mamccrea) [inline stream-analytics-docs/getmetadatapropertyvalue.md:93]:
```suggestion
* This function doesn't work in the Azure portal preview results pane or Visual Studio and VS Code local query testing. It only works when the job is running in the public cloud.
```

**Comment by PRMerger12** [comment]
@an-emma : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by mamccrea** [comment]
#sign-off

**Comment by PRMerger15** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**